controller/lib/core/controller.cpp

```diff
@@ -77,6 +77,7 @@ Controller::Run(Time now, const VentParams &params,
   if (desired_state.is_new_breath) {
     // The "correct" volume at the breath boundary is 0.
     flow_integrator_->NoteExpectedVolume(ml(0));
+    breath_id_++;
```

Comment:
Note that this condition is also met when we transition from an On mode to the Off mode

Reply:
Yup I think that's fine.